config/config.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 const (
-	defaultMsBetweenPoll         = 30000
+	defaultMsBetweenPoll         = 300000
```

Comment:
It would be nice to provide this at a global level as well as a per repo config.